network/src/main/scala/com/linkedin/norbert/network/netty/ClientChannelHandler.scala

```diff
@@ -212,7 +214,10 @@ class ClientStatisticsRequestStrategy(val stats: CachedNetworkStatistics[Node, U
       val nodeP = p.get(n).getOrElse(StatsEntry(0.0, 0, 0))
 
       val nodeMedian = doCalculation(Map(0 -> nodeP),Map(0 -> nodeN))
-      val available = nodeMedian <= clusterMedian * outlierMultiplier + outlierConstant
+      var available = true
```

Comment:
1. Can save the node median computation overheads.
2. Following Scala coding conventions: if possible try to use val instead of var.